tests/roles/run_tests/vars/main.yml

```diff
@@ -125,7 +125,6 @@ verify_actions:
     - "pivoting"
     - "healthcheck"
 pivot_actions:
-    - "ci_test_provision"
```

Comment:
What is this doing? Is it for stopping before pivot to fetch manifests?

Reply:
I removed it to stop the trigger before pivot and that enables to collect the manifest before pivot.